test/TestPkg/src/TestPkg.jl

```diff
@@ -129,30 +129,30 @@ function TaylorIntegration.jetcoeffs!(::Val{f2!}, t::Taylor1{_T}, q::AbstractVec
 end
 
 ex1 = :(function f3!(dq, q, params, t)
-local N = Int(length(q)/2)
-local _eltype_q_ = eltype(q)
-local μ = params
-X = Array{_eltype_q_}(undef, N, N)
-accX = Array{_eltype_q_}(undef, N) #acceleration
-for j in 1:N
-    accX[j] = zero(q[1])
-    dq[j] = q[N+j]
-end
-#compute accelerations
-for j in 1:N
+    local N = Int(length(q)/2)
```

Comment:
Why were the `local` moved inside the `for` loop?

Reply:
I don't [see](https://github.com/PerezHz/TaylorIntegration.jl/blob/628abaf817e26583fcdd6d76e518d8c9bb1b19c9/test/TestPkg/src/TestPkg.jl#L19) that the `local` statements have been moved inside the `for` loop... Maybe changing the indentation led to this interpretation...